### ISSUE TITLE
enumNamespaces overwrite imports

### ISSUE THREAD

**Issue by pronebird** [nicklockwood/SwiftFormat#1581]
Seems like 0.52.9 overwrites the following targeted import:

```
import struct Network.IPv4Address
```

with:

```
import enum Network.IPv4Address
```

Which breaks compilation as you can imagine.

**Comment by nicklockwood** [comment]
D'oh - thanks, I'll get that fixed.

**Comment by guillaumealgis** [comment]
Seeing the same thing with

```
import class MyLibrary.User
```

being replaced with

```
import enum MyLibrary.User
```

(I'm guessing the fix is the same, I'm just signaling that the bug is not limited to `import struct X`)

**Comment by nicklockwood** [comment]
@guillaumealgis I'm having some trouble reproducing this. Can you show me the whole file (or at least the first few lines up to an including some actual code)? I suspect the bug depends on what comes before or after the import.

**Comment by wmorgue** [comment]
Temporary fix:

```swift
// swiftformat:disable enumnamespaces
import struct SwiftUI.Color

// some code...
```

**Comment by nicklockwood** [comment]
@pronebird @guillaumealgis @wmorgue fixed in 0.52.10
